### PR TITLE
[Components with more than 1 service] LPS-190897 Refactor AMImageProcessorImpl component

### PR DESCRIPTION
**Pull request by Valmir-unicap**
Related: https://liferay.atlassian.net/browse/LPS-190897

Here is the thread about unifying the AMImageProcessor and AMProcessor interfaces:
https://liferay.slack.com/archives/CLD39UKM5/p1689884709714769

Correction of PR: https://github.com/liferay-core-infra/liferay-portal/pull/5078

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:stop

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by zhangyan0701** [comment]
ci:test:lima

**Comment by catarineaguiar** [comment]
@Valmir-unicap possible regression:

```
     [exec] com.liferay.document.library.opener.google.drive.test.DLOpenerGoogleDriveManagerTest > testCheckInUploadsAnEmptyFileToGoogle FAILED
     [exec]     com.liferay.portal.kernel.exception.PortalException: java.net.SocketTimeoutException: Read timed out
     [exec]     	at com.liferay.document.library.opener.google.drive.web.internal.DLOpenerGoogleDriveManager.delete(DLOpenerGoogleDriveManager.java:153)
     [exec]     	at com.liferay.document.library.opener.google.drive.web.internal.service.DLOpenerGoogleDriveDLAppServiceWrapper.checkInFileEntry(DLOpenerGoogleDriveDLAppServiceWrapper.java:118)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
     [exec]     	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
```
```
     [exec] com.liferay.document.library.opener.google.drive.test.DLOpenerGoogleDriveManagerTest > testCheckOutUploadsTheFileToGoogle FAILED
     [exec]     java.lang.RuntimeException: com.liferay.document.library.opener.exception.NoSuchFileEntryReferenceException: No DLOpenerFileEntryReference exists with the key {referenceType=GoogleDrive, fileEntryId=42943}
     [exec]     	at com.liferay.document.library.opener.google.drive.web.internal.DLOpenerGoogleDriveManager._getContentFile(DLOpenerGoogleDriveManager.java:374)
     [exec]     	at com.liferay.document.library.opener.google.drive.web.internal.DLOpenerGoogleDriveManager.lambda$checkOut$1(DLOpenerGoogleDriveManager.java:92)
     [exec]     	at com.liferay.document.library.opener.google.drive.web.internal.DLOpenerGoogleDriveFileReference.getContentFile(DLOpenerGoogleDriveFileReference.java:38)
     [exec]     	at com.liferay.document.library.opener.google.drive.test.DLOpenerGoogleDriveManagerTest.lambda$testCheckOutUploadsTheFileToGoogle$1(DLOpenerGoogleDriveManagerTest.java:133)
     [exec]     	at com.liferay.document.library.opener.google.drive.test.DLOpenerGoogleDriveManagerTest.lambda$_test$5(DLOpenerGoogleDriveManagerTest.java:291)
     [exec]     	at com.liferay.document.library.opener.google.drive.test.DLOpenerGoogleDriveManagerTest._test(DLOpenerGoogleDriveManagerTest.java:316)
```
full log [here](https://testray.liferay.com/reports/production/logs/2023-07/test-1-40/test-portal-acceptance-pullrequest(master)/4667/modules-integration-mysql57-jdk8/0/5/jenkins-console.txt.gz?authuser=0)

**Comment by catarineaguiar** [comment]
ci:test:lima

**Comment by catarineaguiar** [comment]
let's check if this test will fail again

File: modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/processor/AMProcessorImpl.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 	property = "model.class.name=com.liferay.portal.kernel.repository.model.FileVersion",
 	service = AMProcessor.class
 )
-public final class AMImageProcessorImpl implements AMProcessor<FileVersion> {
+public final class AMProcessorImpl implements AMProcessor<FileVersion> {
```
Comment (rafaprax) [inline modules/apps/adaptive-media/adaptive-media-image-impl/src/main/java/com/liferay/adaptive/media/image/internal/processor/AMProcessorImpl.java:39]: Hey @Valmir-unicap, please rename this to `AMImageAMProcessor`. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
ci:test:sf

**Comment by Valmir-unicap** [comment]
ci:test:lima

**Comment by Valmir-unicap** [comment]
ci:test:startup

**Comment by Valmir-unicap** [comment]
ci:test:relevant

**Comment by Valmir-unicap** [comment]
ci:test:headless

**Comment by zhangyan0701** [comment]
No regressions found.

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
ci:forward

cc @tinatian

**Pull request closed without merging**